docs/rfc/rfc-5.adoc

```diff
@@ -0,0 +1,121 @@
+:toc: macro
+
+= RFC 5: Emptying and closing wallets
+
+:icons: font
+:numbered:
+toc::[]
+
+== Overview
+
+If a wallet still containing BTC is to be closed,
+the remaining BTC must be transferred to a new wallet if possible.
+
+This applies in the following situations:
+
+- a wallet has fallen below a specified minimum BTC amount
+- a wallet has failed a heartbeat due to losing too many operators
+- a wallet has reached a specified maximum lifetime
+
+When one of these _closure criteria_ is met,
+the wallet changes state to `Emptying`
+and must produce a transaction moving its remaining BTC to a newer wallet.
+
+=== Emptying wallets
+
+When a wallet has began `Emptying`
+the following things change:
+
+- If the wallet was previously a valid deposit wallet,
+it stops being a valid deposit wallet.
+- No redemptions can be made from the wallet.
+- The wallet has 24h to sweep any remaining unswept deposits
+and 48h to transfer its remaining BTC to a newer wallet.
```

Comment:
Any specific reason why you chose the 48h number here?

Reply:
24h to sweep deposits, another 24h to complete the redemption. If we have a different standard timeout we can update this to be 1x and 2x the standard timeout to match.